doc/man3/OpenSSL_version.pod

```diff
@@ -256,6 +275,11 @@ L<crypto(7)>
 The macros and functions described here were added in OpenSSL 3.0,
 except for OPENSSL_VERSION_NUMBER and OpenSSL_version_num().
 
+OPENSSL_VERSION_NUMBER historically did not attempt to follow semantic
```

Comment:
We explicitly do not follow sematic versioning. It's close to it, but it's not the same.

Reply:
Similar to https://github.com/openssl/openssl/pull/28589#discussion_r2357916666, we have needed to add new functionality to fix security bugs.

Reply:
Yeah, so like I said, "... if we *do* <ins>stop adding new public functions</ins> in patch updates ...".  I think we've come full circle 😉